src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs

```diff
@@ -60,70 +61,78 @@
      [comps/Blocker {:banner (if (:locking? @state) "Unlocking..." "Locking...")}])])
 
 
-(defn- render-sidebar [state props refs this ws billing-projects owner? writer?]
-  (let [locked? (get-in ws ["workspace" "isLocked"])
+(defn- render-sidebar [state props refs this ws billing-projects owner? writer? curator?]
+  (let [locked? (get-in ws [:workspace :isLocked])
         status (common/compute-status ws)]
     [:div {:style {:flex "0 0 270px" :paddingRight 30}}
      [comps/StatusLabel {:text (str status
                                  (when (= status "Running")
-                                   (str " (" (get-in ws ["workspaceSubmissionStats" "runningSubmissionsCount"]) ")")))
+                                   (str " (" (get-in ws [:workspaceSubmissionStats :runningSubmissionsCount]) ")")))
```

Comment:
Not in scope for this, but something to think about. I feel like we want our own `js->clj` function that performs an "everything else" -> "idiomatic naming" translation. For example, `"fooBar"` becomes `:foo-bar`, `"i_hate_this_one"` becomes `:i-hate-this-one`, etc.